docs/source/developer/contributing.rst

```diff
@@ -351,15 +351,10 @@ Notes:
    called something else (such as "python3") then parts of the build
    will fail. You may wish to build in a conda environment, or make an
    alias.
--  Some of the packages used in the development environment require
-   Python 3.0 or higher. If you encounter an ImportError during the
-   installation, make sure Python 3.0+ is installed. Also, try using the
-   Python 3.0+ version of ``pip`` or ``pip3 install -e .`` command to
-   install JupyterLab from the forked repository.
 -  If you see an error that says ``Call to 'pkg-config pixman-1 --libs'
    returned exit status 127 while in binding.gyp`` while running the
    ``pip install`` command above, you may be missing packages required
-   by ``canvas``. Please see `Installing Node.js and jlpm section`_
+   by ``canvas``. Please see the `"Installing Node.js and jlpm" section`_
```

Comment:
This breaks the test:

```
/home/runner/work/jupyterlab/jupyterlab/docs/source/developer/contributing.rst:439: ERROR: Unknown target name: ""installing node.js and jlpm" section".
```

I think you will need to use either rename the explicit target in line 292 or use the [`:ref:` syntax](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-ref-role).